packages/block/test/block.spec.ts

```diff
@@ -157,4 +157,73 @@ tape('[Block]: block functions', function (t) {
     }, 'should not throw on DAO HF block with correct extra data')
     st.end()
   })
+
+  t.test(
+    'should set canonical difficulty if I provide a calcDifficultyFromHeader header',
```

Comment:
I think it should rather be avoided to write test descriptions out of the self perspective 😄 , won't let this be a block though of course, just some note.

Reply:
Ah yes you are right 😅  Will consider this too.